app/renderer/components/config_menu.js

```diff
@@ -49,8 +50,25 @@ module.exports = {
         </div>
     </div>
     `,
-
-    methods: {
+   methods: {
+        importTasks() {
+            dialog.showOpenDialog({
+                filters: [{
+                    name: 'json',
+                    extensions: ['json']
+                }]
+            }, (fileNames) =>{
+                if (fileNames === undefined) return;
```

Comment:
if (!filenames) is a better check, as it checks for null as well

Probably you should check that fileName[0] exists as well